db/schema_tables.cc

```diff
@@ -1567,7 +1603,7 @@ static std::vector<data_type> read_arg_types(const query::result_set_row& row, c
     }
 #endif
 
-static shared_ptr<cql3::functions::user_function> create_func(replica::database& db, const query::result_set_row& row) {
+static shared_ptr<cql3::functions::user_function> create_func(replica::database& db, const query::result_set_row& row, lw_shared_ptr<query::result_set> _rs) {
```

Comment:
_ prefix is for members, not arguments.

Better name rs to reflect its role, nobody will guess why the function is receiving both a row and a results set.

Reply:
I've removed this argument, it's no longer necessary